lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -514,19 +514,8 @@
         }
       }
     },
-    "barcelona": {
-      "from": "x86_64",
-      "vendor": "AuthenticAMD",
-      "features": [
-        "mmx",
-        "sse",
-        "sse2",
-        "sse4a",
-        "abm"
-      ]
-    },
```

Comment:
I guess we won't see any of these so this is ok.  Just curious -- was this wrong?

Reply:
Not wrong, but I needed either to complicate the matching algorithm for AMD or find a way to check if all the instructions in `barcelona` (10h family) were compatible with 15h and 17h family. In the end I thought it was old enough so that we can just skip it for the time being.